### PR TITLE
Add Instabug Integration

### PR DESCRIPTION
**Pull request by AbdallahHemdan**
### What does this PR do?
Add integration for Instabug dashboard inside datadog

### Motivation
Allow Datadog users that already use Instabug to have a single source of truth of the data and be able to view the main insights from Instabug dashboard

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, the docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes
Nothing

### CODE REVIEW

File: instabug/README.md
Comment (buraizu) [inline instabug/README.md:16]:
```suggestion
4. Search for `Instabug` to find the Instabug widget in the Apps section of the widget drawer.
```

File: instabug/README.md
Comment (buraizu) [inline instabug/README.md:17]:
```suggestion
5. Click or drag the Instabug widget icon to add it to your dashboard and open the Instabug editor modal.
```

File: instabug/README.md
Comment (buraizu) [inline instabug/README.md:19]:
```suggestion
7. Optionally, give the widget a title.
```

File: instabug/README.md
Comment (buraizu) [inline instabug/README.md:20]:
```suggestion
8. Press **Save** to finish configuring the Datadog dashboard widget.
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thank you for the PR, I've made some suggestions as outlined in our contributing guidelines:
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md

Please let me know if you have any questions.

File: instabug/manifest.json
Comment (emarsha94) [inline instabug/manifest.json:5]:
```suggestion
  "display_on_public_website": true,
```

File: instabug/manifest.json
Comment (emarsha94) [inline instabug/manifest.json:13]: We highly recommend including images like dashboards, graphs, etc

File: instabug/manifest.json
Comment (emarsha94) [inline instabug/manifest.json:15]:
```suggestion
      "Offering::Integration",
      "Supported OS::Linux",
```

File: instabug/images/IMAGES_README.md
Comment (emarsha94) [inline instabug/images/IMAGES_README.md:1]: This file can be removed

**Review by emarsha94 — CHANGES_REQUESTED** [review]

**Comment by emarsha94** [comment]
Hi @AbdallahHemdan - excited for the integration. I left some comments on the PR along with our documentation team. Please let me know if you have questions! 

Additionally, can you please include a logo in .svg format? Thanks!

**Comment by AbdallahHemdan** [comment]
@emarsha94 Thanks for your detailed review, I have applied all the review comments. Our team is working on the media part, I gonna update it once it becomes ready.

File: instabug/README.md
Comment (buraizu) [inline instabug/README.md:4]:
```suggestion
[Instabug][1] helps mobile teams proactively improve the quality of their mobile apps and deliver a high-quality experience to end users. You can use APM, Crash Reporting, and Bug Reporting tools to help optimize your app's performance and improve the end-user experience.
```

**Review by buraizu — APPROVED** [review]
Thank you for implementing those changes, and apologies -- I realize that my suggestions introduced some repetition into the leading paragraph.

This followup suggestion addresses that, but otherwise this looks good to me.

File: .github/CODEOWNERS
Comment (FlorentClarret) [inline .github/CODEOWNERS:48]: Could you also add instabug in [this section](https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L174) so our doc team gets notified when the README, metadata or manifest files are modified?

File: instabug/datadog_checks/instabug/__about__.py
Comment (FlorentClarret) [inline instabug/datadog_checks/instabug/__about__.py:2]:
```suggestion
__version__ = '1.0.0'
```

This will be the first released version 🙂

**Review by FlorentClarret — COMMENTED** [review]
Hi! I left some minor comments. I'll review it again once the python files will be up to date!

File: instabug/README.md
Comment (emarsha94) [reply instabug/README.md:29]: @AbdallahHemdan bumping this!

File: instabug/manifest.json
Comment (emarsha94) [reply instabug/manifest.json:13]: Hi @AbdallahHemdan - saw you marked this as resolved. We highly recommend images be included in the offering. Are you able to include images here?

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: instabug/README.md
Comment (AbdallahHemdan) [reply instabug/README.md:29]: @emarsha94 I can see that all the integrations on the master branch use `Datadog Support`, do we need to change that ?

**Comment by AbdallahHemdan** [comment]
@FlorentClarret @emarsha94 We have added all the media files, could you please check again and let me know if there is anything missing?

**Comment by anawishnoff** [comment]
Hi @AbdallahHemdan, our team emailed over a PDF with feedback on your app last month. Once that's addressed, we can continue to review and merge this PR. Thanks!

**Comment by AbdallahHemdan** [comment]
@anawishnoff Thanks for your fast response, I think we are done with applying all the `required` comments you sent 👍🏻

File: instabug/README.md
Comment (emarsha94) [reply instabug/README.md:30]: Hi @AbdallahHemdan - yes, this will need to be changed to Instabug's support team email alias. Thanks!

File: instabug/manifest.json
Comment (emarsha94) [inline instabug/manifest.json:75]: This needs to be an email alias instead of a specific person. Thanks!

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Once the support emails are changed, we will just need to update the Changelog date to reflect the release date which I will request before we merge! Thanks!

**Comment by AbdallahHemdan** [comment]
@emarsha94 I have applied all the review comments, please check and let me know if you have any other comments 👍🏻 
Regarding updating the Changelog date to reflect the release date, I think we can do that after aligning the merge for the marketplace PR, once confirmed I will let you know about the release day 🙏🏻

File: .github/CODEOWNERS
Comment (emarsha94) [inline .github/CODEOWNERS:72]: Please add @DataDog/marketplace-review to the codeowners file

File: instabug/CHANGELOG.md
Comment (emarsha94) [inline instabug/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2023-01-24
```

**Review by emarsha94 — APPROVED** [review]
Two minor changes:
1. Add @DataDog/marketplace-review to the codeowners file
2. Changelog date

Thanks for all your hard work on this!

**Comment by eho1307** [comment]
Spoke over email but adding it here for visibility. This PR includes files needed for an agent based integration. Since this is an app, please close this PR and re-open a new one with the correct files.

**Pull request closed without merging**
